### PR TITLE
Add Web Budget API

### PR DESCRIPTION
**Pull request by jpmedley**
So that I can document this feature for Chrome and Opera: https://www.chromestatus.com/features/5691190548627456

### CODE REVIEW

**Comment by jpmedley** [comment]
Also, if someone can tell me where the spec2 template is located, I would be very appreciative.

**Comment by jpmedley** [comment]
I found spec2. It's in a [PR that hasn't been merged](https://github.com/mozilla/kumascript/commit/b1d87c9d210b4bdd0a97f86baacad0efcfbd863a).

**Comment by Elchi3** [comment]
I don't see Web Budget API in spec2 in the other PRs you've opened.

**Comment by jpmedley** [comment]
I can't commit to spec2 or open PRs against it.

File: macros/SpecName.ejs [hunk 1]
```diff
@@ -1138,4 +1142,4 @@ if ($2) {
 }
 
 
-%><%- result %>
+%><%- result %>
```
Comment (jpmedley) [inline macros/SpecName.ejs]: It's a byproduct of adding lines higher up. The initial - and + aren't part of the file. They're git's identifiers for what's changed.

Comment (jpmedley) [inline macros/SpecName.ejs]: This looks to me like a false positive. The only thing different is what line it appears on.

**Pull request merged**